docker/run/Dockerfile

```diff
@@ -43,14 +44,19 @@ ENV \
     SBT_VERSION=0.13.13-2 \
     YARN_VERSION=1.17.3 \
     # Scanner versions.
-    SCANCODE_VERSION=2.9.7
+    SCANCODE_VERSION=2.9.7 \
+    # Installation directories.
+    FLUTTER_HOME=/opt/flutter
+
+ENV PATH="$PATH:$FLUTTER_HOME/bin:$FLUTTER_HOME/bin/cache/dart-sdk/bin"
 
 # Apt install commands.
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         # Install general tools required by this Dockefile.
         curl \
         openssh-client \
+        lib32stdc++6 \
```

Comment:
Should we sort this alphabetically before "o", or is your plan to make libs go last and command line tools frist?

Reply:
No, I just didn't sort it. Done.